src/filesys.cpp

```diff
@@ -740,5 +744,23 @@ bool Rename(const std::string &from, const std::string &to)
 	return rename(from.c_str(), to.c_str()) == 0;
 }
 
+bool bad_char(const char c)
+{
+	return !(isalnum(c) || c == '-' || c == '_' || c == ' ');
+}
+
+std::string SanitizeFilename(std::string input)
```

Comment:
input is passed by value so it should be the same as passing a const and returning a new value